nifi-assembly/pom.xml

```diff
@@ -19,6 +19,10 @@ language governing permissions and limitations under the License. -->
     <artifactId>nifi-assembly</artifactId>
     <packaging>pom</packaging>
     <description>This is the assembly Apache NiFi</description>
+    <!--Add a blank run.as in case RPM isn't being used -->
+    <properties>
+        <nifi.run.as />
```

Comment:
@jfrazee just found time to chase this and it look good. Give me a few more minutes and a new iteration will be available.